src/main/java/TaskList.java

```diff
@@ -0,0 +1,91 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    List<Task> list;
+    int listSize;
```

Comment:
Should these variables be private (with getter methods to retrieve these fields in the class)? 🤔